recipe/build.sh

```diff
@@ -51,3 +51,16 @@ chmod -R o-w "${SRC_DIR}"
 # https://rt.perl.org/Public/Bug/Display.html?id=128020
 # make test
 make install
+
+# Replace hard-coded BUILD_PREFIX by value from env as CC, CFLAGS etc need to be properly set to be usable by ExtUtils::MakeMaker module
+sed -i.bak "s|\(local \*.*\)|use File::Basename; my \$compilerroot = dirname(\$ENV{'CC'}).\"\/..\/\"; \1|g" $PREFIX/lib/*/*/Config_heavy.pl
```

Comment:
@abretaud I know its more work, but can we use a patch here, I think this would be more readable.

Reply:
Hum, yep it should be possible for the 2 first sed commands, for the others it's modifying variable content so I'll have to leave them
I'll look at updating to 5.26.3 too
thanks for looking at it!